introduction.tex

```diff
@@ -8,24 +8,24 @@ \section{Introduction}\label{section:introduction}
+A user who wishes to acquire a target cryptocurrency first has to form a specially crafted address called the \emph{burn address}, which encodes among other things her receiving address on the target blockchain as metadata. The user then sends an amount of the source cryptocurrency to that address, burning this amount in the source blockchain. She then proceeds to submit a proof of this burn on the target blockchain, where the burn is verified and the user is credited an equivalent amount of target cryptocurrency on her receiving address. The equivalent amount can be determined based on the macroeconomic model of the target cryptocurrency and is beyond the scope of this work. Upon burning her coins the user can generate a proof-of-burn which is a string which can be verified by any interested party given the metadata. The verification of the proof of the burn can happen in either a centralized manner which is lighter on computation but requires a federation, or in a decentralized manner using Non-Interactive Proofs of Proof-of-Work (NIPoPoWs).
 
 \noindent
 \textbf{Properties.}
-Like the rest of the system, proof-of-burn for money creation must be decentralized, i.e., there cannot be any ``committees'' or individuals verifying that proof-of-burn transactions took place and vouching for them. Additionally, target blockchain miners must remain connected only to the target blockchain network, hence we cannot mandate that every miner connects to every other network that we wish to burn funds from. In addition, the system should be uncensorable by source blockchain miners in case any of them decide that they do not agree with it. Finally the system should be easy to use, meaning that a user should be able to complete this procedure by using his existing wallet for the source cryptocurrency. To summarise, these are the properties that we desire such a system to satisfy:
+Like the rest of the system, proof-of-burn for money creation must be decentralized, i.e., there cannot be any ``committees'' or individuals verifying that proof-of-burn transactions took place and vouching for them. Additionally, target blockchain miners must remain connected only to the target blockchain network, hence we cannot mandate that every miner connects to every other network that we wish to burn funds from. In addition, the system should be uncensorable by source blockchain miners in case any of them decide that they do not agree with it. Finally the system should be easy to use, meaning that a user should be able to complete this procedure by using her existing wallet for the source cryptocurrency. To summarise, these are the properties that we desire such a system to satisfy:
```

Comment:
Be consistent in spelling proof-of-burn as "Proof of Burn"

Reply:
"there cannot be any ``committees'' or individuals verifying that proof-of-burn transactions took place and vouching for them" we kind of contradict that when we speak about the federation, let's just say that it is possible to do without a committee

Reply:
"target blockchain miners must remain connected only to the target blockchain network" again we contradict this when speaking about direct observation, let's just say that it's possible to avoid connecting...

Reply:
Shorten:
* decide that they do not agree with it -> decide they do not agree with it
* meaning that a user should be able to -> meaning a user can
* these are the properties that we desire such a system to satisfy -> the desirable properties are as follows

Reply:
these fixes are applied to #32 as the sections are ultimately moved